server/events/views.py

```diff
@@ -0,0 +1,3 @@
+from django.shortcuts import render
```

Comment:
Should this file be empty?

Reply:
Notifications views and notifications models were two separate issues. Figured I'd do the events views when the views issue was opened.

Reply:
Oh so this just got initialized with the app. Gotcha. Thanks!